powershell/VstsTaskSdk/FindFunctions.ps1

```diff
@@ -11,7 +11,7 @@ Directory to search.
 .PARAMETER LegacyPattern
 Proprietary pattern format. The LiteralDirectory parameter is used to root any unrooted patterns.
 
-Separate multiple patterns using ";".
+Separate multiple patterns using ";". Escape actual ";" in the path by using ";;".
```

Comment:
I had made this edit previously to the docs but forgot to pull it back into the comment in the source code. Since the docs are generated from the comments, it was getting deleted from the docs after I regenerated.